WinUIGallery/Controls/PageHeader.xaml

```diff
@@ -75,72 +77,80 @@
+                                            <FontIcon VerticalAlignment="Center"
+                                                      FontSize="14"
+                                                      Foreground="{ThemeResource AccentTextFillColorPrimaryBrush}"
+                                                      Glyph="&#xE946;" />
+                                        </Button>
                                     </StackPanel>
-                                    <HyperlinkButton
-                                        x:Name="ControlSourceLink"
-                                        Margin="-12,4,0,0"
-                                        ToolTipService.ToolTip="{x:Bind ControlSourceLink.NavigateUri, Mode=OneWay}">
+                                    <HyperlinkButton x:Name="ControlSourceLink"
+                                                     HorizontalAlignment="Stretch"
+                                                     HorizontalContentAlignment="Left"
+                                                     Margin="-12,4,-12,0"
+                                                     ToolTipService.ToolTip="{x:Bind ControlSourceLink.NavigateUri, Mode=OneWay}">
                                         <TextBlock Text="{x:Bind Item.Title}" />
                                     </HyperlinkButton>
                                 </StackPanel>
 
-                                <StackPanel Orientation="Horizontal" Spacing="8">
-                                    <TextBlock
-                                        VerticalAlignment="Center"
-                                        Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                                        Style="{StaticResource CaptionTextBlockStyle}"
-                                        Text="Sample page source code" />
-                                    <FontIcon
-                                        VerticalAlignment="Center"
-                                        FontSize="14"
-                                        Foreground="{ThemeResource AccentTextFillColorPrimaryBrush}"
-                                        Glyph="&#xE946;"
-                                        ToolTipService.ToolTip="Source code of this sample page in the WinUI Gallery repository" />
+                                <StackPanel Orientation="Horizontal"
+                                            Spacing="8">
+                                    <TextBlock VerticalAlignment="Center"
+                                               Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                               Style="{StaticResource CaptionTextBlockStyle}"
+                                               Text="Sample page source code" />
+                                    <Button Padding="6,5,6,6"
```

Comment:
Hmm these look a bit weird.. and not something you'd see across Windows. If it looks like a button, I'd expect something to happen when clicking on it?

Could we use `SubtleButtonStyle` (see source [here,](https://github.com/microsoft/PowerToys/blob/3e133ae8bdd63003fe3160a036777147fbc25c20/src/settings-ui/Settings.UI/SettingsXAML/Styles/Button.xaml#L57) should be added to the platform IMO) so atleast it doesn't look like a button but would provide hover/pressed states. I'm happy to add the style here as well if you want me to!

Reply:
Turns out there was already an unused instance of subtle button style already. I moved it to app.xaml so that it can be consumed by other user controls. 

I can make changes to other user controls in a separate PR.